support.js

```diff
@@ -169,21 +126,31 @@ async function storeSubmission(triples, submissionGraph, fileGraph, authenticati
 
     //update created-at/modified-at for submission
     await update(`
-      ${PREFIXES}
+      ${env.PREFIXES}
       INSERT DATA {
         GRAPH ${sparqlEscapeUri(submissionGraph)} {
             ${sparqlEscapeUri(extractSubmissionUrl(triples))}  dct:created  ${sparqlEscapeDateTime(new Date())}.
             ${sparqlEscapeUri(extractSubmissionUrl(triples))}  dct:modified ${sparqlEscapeDateTime(new Date())}.
         }
       }
     `);
-    return taskUri;
+
+    await jobsAndTasks.automaticSubmissionTaskSuccess(submissionGraph, automaticSubmissionTaskUri, jobUri, remoteDataUri);
+
+    return jobUri;
   } catch (e) {
-    console.error('Something went wrong during the storage of submission');
-    console.error(e);
+    console.error(`Something went wrong during the storage of submission ${meldingUri}. This is monitored via task ${automaticSubmissionTaskUri}.`);
+    console.error(e.message);
     console.info('Cleaning credentials');
-    await cleanCredentials(authenticationConfiguration);
-    if (newAuthConf.newAuthConf) {
+    const errorUri = await sendErrorAlert({
+      message: `Something went wrong during the storage of submission ${meldingUri}. This is monitored via task ${automaticSubmissionTaskUri}.`,
+      detail: e.message,
+    });
+    await jobsAndTasks.automaticSubmissionTaskFail(submissionGraph, automaticSubmissionTaskUri, jobUri, errorUri);
+    e.alreadyStoredError = true;
+    if (authenticationConfiguration)
```

Comment:
See previous comment about not really wanting to change original code at this point. This could use some reworking, but can we do this in a later stage with separate library and decent code clean-up?